zebra/Makefile.am

```diff
@@ -85,7 +85,7 @@ EXTRA_DIST = if_ioctl.c if_ioctl_solaris.c if_netlink.c \
 client : client_main.o ../lib/libfrr.la
 	$(CC) -g -o client client_main.o ../liblzebra.la $(LIBS) $(LIB_IPV6)
 
-quaggaconfdir = $(sysconfdir)
+frrconfdir = $(sysconfdir)
```

Comment:
I didn't find any use for it, kept it just in case I missed something, and also in the spirit of the whole batch which was purely a "rename"